migrations/.common.js

```diff
@@ -4,14 +4,20 @@ let kue = require('kue')
 let queue = kue.createQueue(Object.assign(config.kue, { redis: config.redis }))
 
 let es = require('elasticsearch')
+let port = config.elasticsearch.port
+if (!port || port.length < 2) {
+  port = ''
+} else {
+  port = `:${port}`
+}
```

Comment:
Wouldn't it be better to check the port with `parseInt()` instead of checking length?